vep/__init__.py

```diff
@@ -279,15 +281,45 @@ def install_build_tools(self, pip):
             else:
                 pip('install', "%s==%s" % (tool, version), _out=print_line)
 
+    def _pip_requirements_filename(self, path=''):
+        """
+        Returns filename of pip requirements file, & verifies that the file exists.
+
+        :param path: Path to dir to search. Default it current dir.
+        :type path: str
+        :raises VEPackagerError
+        :return: Filename of pip requirements
+        :rtype: str
+        """
+        if self.args.pip_requirements:
+            filenames = list(self.args.pip_requirements)
+        else:
+            filenames = list(DEFAULT_REQUIREMENTS_FILES)
```

Comment:
Why not just make this a list from the start? Or just iterate over a tuple?

Reply:
Also, wouldn't it be better to set `DEFAULT_REQUIREMENTS_FILES` as the default value in `argparse` so you don't need to do this?

Reply:
Because `--pip-requirements` should be either a string or nothing. A CLI user shouldn’t be able to supply several requirements files, so it shouldn’t be a list. Yeah, I _could_ do that, but it would be _weird_.

Reply:
Sounds good. Then my only minor point would be converting a tuple into a list for the defaults. I don't really see the point.